src/Symfony/Component/EventDispatcher/InvokableEventSubscriberTrait.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\EventDispatcher;
+
+/**
+ * Helper trait for single-event subscription.
+ *
+ * By default, the event name will be inferred from the FQCN of
+ * the event parameter, but you will can explicitly define it
+ * by declaring the static property `$listenerEvent` as well as
+ * the static property `$listenerPriority` if the subscription
+ * needs a specific priority.
```

Comment:
what about this ?
`By default, the type of the first argument of the "__invoke" method defines which events should be listened for. This can be overridden by defining a "protected static $listenerEvent" property. A "protected static $listenerPriority" property can also be defined to change the priority of the listener. Don't forget to declare "implements EventSubscriberInterface" when using this trait.`

Reply:
Thank you.

Is there any reason to define these properties as `protected`? I think `private` will work too.

Reply:
private would contradict with using `static::` in the implementation to me